server/test/unit/server/webservices/views/test_schedule.py

```diff
@@ -0,0 +1,60 @@
+import unittest
```

Comment:
@ipanova, I missed these tests on the first go around. Please be sure to have a look at this file too!